packages/shared/locales/en.json

```diff
@@ -1751,7 +1751,7 @@
             }
         },
         "syncing": "Please wait until synchronization is finished to change wallets.",
-        "transferring": "Please wait until all transactions are completed to change wallets.",
+        "transferring": "Please wait until all transactions are completed.",
```

Comment:
@begonaalvarezd Can we just change this string like this ?
The removed part makes it sound like it is used in some other place for a different purpose. Wdyt ?

Reply:
I've removed the last part because that string is not used anywhere, looks like a leftover so i reused it

Reply:
i would say its ok, now its more generic